lib/pharos/host/ubuntu/scripts/install-kube-packages.sh

```diff
@@ -7,6 +7,9 @@ apt-mark unhold kubelet kubectl kubeadm
 apt-get install -y kubelet=${KUBE_VERSION}-00 kubectl=${KUBE_VERSION}-00 kubeadm=${KUBEADM_VERSION}-00
 apt-mark hold kubelet kubectl kubeadm
 
+# use KUBELET_EXTRA_ARGS from /etc/systemd/system/kubelet.service.d/11-pharos.conf instead
+sed -i 's/^KUBELET_EXTRA_ARGS=/#\0/' /etc/default/kubelet
```

Comment:
Maybe `ConfigureKubelet` should be writing the `/etc/default/kubelet` instead, but that's annoying since it varies by OS.